vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -134,21 +134,19 @@ public String toString() {
     @SerializedName("duration")
     public int mDuration;
 
-    /**
-     * Deprecated in favor of {@link Play}
-     * and the multiple {@link VideoFile}
-     * that it contains.
-     */
-    @Deprecated
-    @SerializedName("files")
-    public ArrayList<VideoFile> mVideoFiles;
-
     @SerializedName("width")
     public int mWidth;
 
     @SerializedName("height")
     public int mHeight;
 
+    /**
+     * The Embed field is in the process of moving on to the {@link #mPlay} field
```

Comment:
Awesome docs.  Maybe add `@deprecated` annotation to the start.  Note; capital-D `@Deprecated` marks method as deprecated, and lowercase `@deprecated` shows javadocs related to the depreciation of the method..